### ISSUE TITLE
Sorting with direction breaks the URL (options hash is passed to ES)

### ISSUE THREAD

**Issue by matsko** [karmi/retire#305]
I've been trying to setup a descending search with the following code:

``` ruby
field = 'id'
direction = 'desc'
search_str = '*' #trying to find everything
options = {}
options[:sort]=field.to_s + ' ' + direction.to_s
options[:page] ||= 1
options[:per_page] = options[:limit] if options[:limit]
Movie.search(search_str,options)
```

And when I look in the log to take a look at the curl command I find:

```
curl -X GET "http://localhost:9200/movies/movie/_search?page=1&sort=id+desc&pretty=true" -d '{"query":{"query_string":{"query":"*"}},"sort":[{"id":"desc"}]}'
```

So it looks like it picks up the column + direction values properly, but it seems to **escape** the URL parameter.

Turns out that the space doesn't work with elasticsearch 0.18.5. I tried to remove the "sort=" part from the curl URL and it worked:

```
curl -X GET "http://localhost:9200/movies/movie/_search?page=1&pretty=true" -d '{"query":{"query_string":{"query":"*"}},"sort":[{"id":"desc"}]}'
```

Could the sort= part of the curl request be removed? Will this cause any problems?

**Comment by karmi** [comment]
That's a bug where we pass the search options (implemented in d764c0d) to ES. For the time being, better use the blockish DSL syntax to perform sorting...

**Comment by romanbsd** [comment]
It's solved in #334

**Comment by karmi** [comment]
@matsko Closed by https://github.com/karmi/tire/commit/351d0c4b506f1017e910164f8a43ddc0679fde4d. Could you try?
